frontend/nexusInteraction.cpp

```diff
@@ -420,4 +424,185 @@ void MainWindow::showStatus(qreal x, qreal y, QString title)
     else
         message = QString::number(x) + ", " + QString::number(y);
     statusBar()->showMessage("Run " + title + ": " + message);
+}
+
+void MainWindow::handleSpectraCharting(HttpRequestWorker *worker)
+{
+    auto *window = new QWidget;
+    auto *chart = new QChart();
+    auto *chartView = new ChartView(chart, window);
+
+    QString msg;
+    if (worker->error_type == QNetworkReply::NoError)
+    {
+        QString field = "Spectra ";
+        QString runs;
+        bool init = true;
+        foreach (const auto &run, worker->json_array)
+        {
+            field += init ? run.toString() : nullptr;
+            if (init)
+            {
+                init = false;
+                continue;
+            }
+            auto runArray = run.toArray();
+            // For each plot point
+            auto *series = new QLineSeries();
+            // Set dateSeries ID
+            QString name = runArray.first().toString();
+            runs += name + ", ";
+            series->setName(name);
+            runArray.removeFirst();
+
+            foreach (const auto &dataPair, runArray)
+            {
+                auto dataPairArray = dataPair.toArray();
+                series->append(dataPairArray[0].toDouble(), dataPairArray[1].toDouble());
+            }
+            chart->addSeries(series);
+        }
+        chart->createDefaultAxes();
+        chart->axes(Qt::Horizontal)[0]->setTitleText("Time of flight");
```

Comment:
```suggestion
        chart->axes(Qt::Horizontal)[0]->setTitleText("Time of flight, &#181;s");
```
The `&#181;` should give the Greek mu.